pegasus/sites.v3/code.org/views/congrats.haml

```diff
@@ -24,8 +18,8 @@
         %img{:src=>"/images/fit-540/#{certificate_template_for(cert_script)}", :width=>"100%"}
 
     #certificate.col-50
-      = view :congrats_certificate
-      = view :congrats_share, cert_id:cert_id, facebook:facebook, twitter:twitter
+      = view :csf_congrats_certificate, course: course
+      = view :congrats_share, facebook:facebook, twitter:twitter
 
   %div{:style=>'clear: both; height:30px'}
   %h1= I18n.t(:congrats_next_tutorials_title)
```

Comment:
it would be nice to use a map, like `recommended_courses` above, in place of `"#{course}_next"` a few lines below this comment, to make it easier to find where `course4_next` is referenced from.